templates/openstack/managed.html

```diff
@@ -1,725 +1,610 @@
               ) | safe
-            }}
-            <h4 class="p-heading-icon__title">Ebook</h4>
+            }}      
           </div>
-          <h3 class="p-heading--four"><a class="p-link--external" href="https://ubuntu.com/engage/charmed-openstack-adoption-whitepaper" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Managed cloud', 'eventLabel' : 'A guide to a successful OpenStack adoption and deployment', 'eventValue' : undefined });">A guide to a successful OpenStack adoption and deployment
-</a></h3>
-        </div>
       </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-deep is-bordered">
-  <div class="row">
-    <div class="col-7">
-      <h2>Security first &mdash; trusted by the OpenStack majority</h2>
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <p>Canonical, a founding member of OpenStack, provides security updates for thousands of production OpenStack clouds worldwide.</p>
-      <p>We offer multi-year maintenance for stable cloud deployments and upgrades for those who want to move to newer versions of OpenStack when they are released.</p>
-      <p>The OpenStack super-users who run the largest and most complex Openstack deployments choose Ubuntu because of our quality and security. Peace of mind comes from working with the people who support the widest range of cloud architectures in the widest range of business sectors and geographies.</p>
-      <p>Canonical will help you meet telco, finance and government sector audit and compliance standards, with kernel live patching, optional FIPS compliance, and a range of security and monitoring integration points.</p>
-    </div>
-    <div class="col-3 col-start-large-9">
-      {% include "openstack/shared/_openstack-pie-chart.html" %}
-    </div>
-  </div>
-</section>
-<section class="p-strip--light is-deep is-bordered">
+<section class="p-strip">
   <div class="row">
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
-      <h2>Managed private cloud, a perfect fit for multi-cloud environments</h2>
-      <p>We recommend a multi-cloud operations strategy for every company running applications at scale. The combination of public clouds and efficient, automated private infrastructure, ideally with outsourced and standardised operations under the hood, offers the best mix of latency and price across both fixed and variable workloads.
-        <a href="/engage/multi-cloud-guide">We understand why no one type of cloud architecture or infrastructure is ideal for everyone</a>
+      <p>While public clouds provide immediate access to the infrastructure-as-a-service (IaaS) offering, they are not economical in the long term and at scale. At the same time, implementing a private cloud comes with a cost.</p>
+      <p>
+        What if:
+        <ul>
+          <li>Your public cloud costs are growing with a trajectory you cannot afford?</li>
```

Comment:
```suggestion
          <li class="p-list__item is-ticked">Your public cloud costs are growing with a trajectory you cannot afford?</li>
```

Reply:
I think in this case tick icons aren't appropriate - they indicate something positive, but this is a list problems/negative things.

Reply:
> I think in this case tick icons aren't appropriate - they indicate something positive, but this is a list problems/negative things.

@sowasred2012 yep, right. That makes sense!